lib/Zef/Service/Shell/Test.rakumod

```diff
@@ -62,7 +62,7 @@ class Zef::Service::Shell::Test does Tester {
         method test(IO() $path, Str :@includes, Supplier :$stdout, Supplier :$stderr --> Bool:D)
 
     Test the files ending in C<.rakutest> C<.t6> or C<.t> in the C<t/> directory of the given C<$path> using the
-    provided C<@includes> (e.g. C</foo/bar> or C<inst#/foo/bar>) via the C<raku> command. A C<Supplier> can be
+    provided C<@includes> (e.g. C</foo/bar> or C<inst#/foo/bar>) via the C<prove> command. A C<Supplier> can be
```

Comment:
Changing this bit of documentation was a thinko - https://github.com/ugexe/zef/pull/561